deploy/helm/values.yaml

```diff
@@ -519,12 +519,40 @@ autoupdate:
 
 kube-state-metrics:
   enabled: true
+  service:
+    annotations:       
+      opencost.scrape: "true"
+
+  prometheusScrape: false  
+
+  nodeSelector:
+    kubernetes.io/os: linux
+    kubernetes.io/arch: amd64
+
+prometheus-node-exporter:  
+  service:
+    port: 9101
+    targetPort: 9101
+    annotations:
+      opencost.scrape: "true"     
+      prometheus.io/scrape: "false"
```

Comment:
Is this needed? isn't this covered by `prometheusScrape: false`?

Reply:
node exporter doesn't have such option

Reply:
left over, remove that from values.yaml. Good catch